RecoPixelVertexing/PixelTriplets/src/CAGraph.h

```diff
@@ -6,14 +6,19 @@
 #include <vector>
 
 struct CALayer {
-  CALayer(const std::string &layerName, std::size_t numberOfHits) : theName(layerName) {
+  CALayer(const std::string &layerName, const int seqNum, std::size_t numberOfHits)
+      : theName(layerName), theSeqNum(seqNum) {
     isOuterHitOfCell.resize(numberOfHits);
   }
 
   bool operator==(const std::string &otherString) { return otherString == theName; }
```

Comment:
We can keep the changes from the previous suggestion but would need to add `const` after the function declaration here. The == operator is used in other parts of the CA code, but it seems to work properly also after making this change (testing with workflow 10824.1)

Reply:
Whatever it works. Removing the const looks safer to me, but if you have already verified that modifying the == operator works as well I let it to you to decide.